Library/Formula/cf.rb

```diff
@@ -0,0 +1,20 @@
+class Cf < Formula
+  homepage "http://ee.lbl.gov"
+  url "ftp://ee.lbl.gov/cf-1.2.5.tar.gz"
+  sha1 "0ef0b03c1ea7221d75dac0ce110fd677e1f0182a"
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    bin.mkdir
+    system "make", "install"
+    mkdir_p (man / "man1")
+    system "make", "install-man"
+  end
+
+  test do
+    # Prints "Jan 19 16:35:44"
+    (testpath / "test.txt").write("1074558944")
+    system "#{bin}/cf", (testpath / "test.txt")
```

Comment:
Might be neat if we can actually ` assert_match ` this to the expected readout of ` "Jan 19 16:35:44" `. Just an idea though.